examples/step-24/step-24.cc

```diff
@@ -267,7 +257,7 @@ namespace Step24
   {
     const Point<dim> center;
     GridGenerator::hyper_ball(triangulation, center, 1.);
-    triangulation.refine_global(7);
+    triangulation.refine_global(8);
```

Comment:
Why do you change the number of refinements? Is this consistent with the output?

Reply:
No, this was a mistake: I changed it to check something but didn't mean to merge it. I will fix this.